walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,18 +15,31 @@
 package com.google.sps;
```

Comment:
Overall, just wondering if I'm on the right track. I'm going through each of the tests in the test file and writing to this file what I need to create as a collection to pass each test. I'm struggling with how to access certain variables or if certain checks should have embedded if statements.

Reply:
Yep, on the right track for trying to solve each test one by one, that's definitely one valid approach.  You could for example do multiple separate solutions, one per test.  Then later on combine them into a single cohesive solution.

Another approach would be to build one big solution off the bat, then adjust the solution incrementally to pass any failing tests.